rln-js/index.html

```diff
@@ -297,6 +297,12 @@ <h4 class="mu1">Messages</h4>
         updateFields();
       };
 
+      const changeEndianess = (hex) => {
+        const bytes = ethers.utils.arrayify(hex);
+        const reversed = bytes.reverse();
+        return ethers.utils.hexlify(reversed);
+      };
+
```

Comment:
This is a lot of manipulation: Uint8Array -> hex string -> bytes (Uint8Array) -> reverse -> hex string.

I would recommend instead:  Uint8Array -> Bigint
and use the bigint in etherjs API moving forward.

You can do that using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DataView/getBigUint64

Reply:
Addressed.

Figured a shorter way:
Uint8Array -> reverse -> pass to ethersjs